src/flint/fmpz.jl

```diff
@@ -85,6 +85,44 @@ ifelse(stop >= start, stop, start - one(ZZRingElem))
 
 Base.:(:)(a::ZZRingElem, b::ZZRingElem) = ZZRingElemUnitRange(a, b)
```

Comment:
It's a bit unfortunate that you put your new code inside the code for `ZZRingElemUnitRange`. Better to move it to be completely after the `ZZRingElemUnitRange` stuff